Sources/Sentry/SentryClient.m

```diff
@@ -184,6 +194,8 @@ - (SentryEvent *_Nullable)prepareEvent:(SentryEvent *)event withScope:(SentrySco
         event = [scope applyToEvent:event maxBreadcrumb:self.options.maxBreadcrumbs];
     }
 
+    event.debugMeta = [self.debugMetaBuilder buildDebugMeta];
```

Comment:
`H`: This should only be done if `attachStacktrace` option is true we don't need to send it otherwise.

Reply:
Very good catch thanks.